version/version.go

```diff
@@ -0,0 +1,16 @@
+package version
```

Comment:
Well, the logic could actually go to `github` though, considering `command` depends on `github` but not the other way around.